### ISSUE TITLE
Add All Genders

### ISSUE THREAD

**Issue by greenreport** [google/fonts#5762]
**Font Project Git Repo URL:**

https://github.com/greenreport/All-Genders

**Super short description of the Font Family:**

Arabic is a grammatically gendered language. However, the male case is often used by default, even in job advertisements that are in fact aimed at all jobseekers. In order to raise awareness on the importance of gender inclusiveness in communication, particularly in the world of work, UN Women and creative agency Serviceplan Health & Life are partnering in an advocacy campaign proposing a new Arabic character: the “All-Genders Sign” which makes it possible to include everyone.

![ServiceplanHealth_Life_UNWomen_AllGendersSign](https://user-images.githubusercontent.com/119879285/211740757-c8f57323-dfd8-4b81-a136-6ea7b5ed8335.jpg)

**Requirements:**

I understand that Google Fonts will publish only fonts that matches its requirements, and I can confirm the project meets them (by ticking the cases, or putting x between the square brackets in text mode):

- [x] The entire font project is licensed under the [OFL](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL)
- [x] There is no "Reserved Font Name" mention in the OFL license
- [x] Family name is unique according to namecheck.fontdata.com
- [x] The font supports at least [GF Latin Core glyphset](https://github.com/googlefonts/glyphsets/tree/main/GF_glyphsets/Latin/)
- [x] The repository respect the [upstream repo structure](https://googlefonts.github.io/gf-guide/upstream.html)
- [x] The source files are available in the repository
- [x] I read, agree and comply with the full [contributing requirements](https://googlefonts.github.io/gf-guide/onboarding.html)
- [x] I am the author of the font or the author agreed to this submission
- [x] I will maintain the repository

**Image:**

Attach here a pic or a screenshot of the font; one is enough, it can be few letters, to give a quick overview.

**Comment by m4rc1e** [comment]
Thanks for the submission.

I agree with all of John's points. The all gender signed has also been mapped to some unicodes used for punctuation such as U+0022 (quotedbl) so passages of English text will look strange.

@greenreport I see Rassam Abdo is the author of this font. What was your involvement? it doesn't look like Rassam releases his fonts under the OFL license.

**Comment by greenreport** [comment]
> 



> Thanks for the submission.
> 
> I agree with all of John's points. The all gender signed has also been mapped to some unicodes used for punctuation such as U+0022 (quotedbl) so passages of English text will look strange.
> 
> @greenreport I see Rassam Abdo is the author of this font. What was your involvement? it doesn't look like Rassam releases his fonts under the OFL license.

That is correct. Mr Rassam is the author of the font. I was commissioned by the Serviceplan Group to realise the font in a technical way. The unicode mapping of the "All-Gender Sign" was already done.

The Serviceplan Group was commissioned by UN Women. The concept is by Serviceplan Group. Mr. Rassam is the designer/author of the font. He knows about the submission to Google Fonts and the OFL licensing.

I am taking care of the technical details and submission it self in this case.

https://arabstates.unwomen.org/en/stories/news/2022/03/gender-equality-in-the-workplace-starts-with-a-job-ad

https://www.serviceplan.com/en/news/unwomen-all-genders-sign.html

**Comment by greenreport** [comment]
@m4rc1e 

Guess, we have a solution for the Unicode mapping. The "All Gender Sign" is now a "liga" feature. Thank you for the suggestion. Now you can reach the character with following charakter combination/ligature:

Arabic: "٭" and "ة" (U+066D and U+0629),  asterisk-arabic and teh marbuta
Latin: "*" and "a" (U+002A and U+0061), asterisk and the lower case "a"
![Bildschirmfoto 2023-01-13 um 08 17 43](https://user-images.githubusercontent.com/119879285/212260548-ccd068a8-89e0-465a-b354-639cd9b42428.png)

**Comment by davelab6** [comment]
@greenreport thanks for proposing this project for inclusion. The connection to a government agency and what is now an unusual OpenType feature, mean we'll take quite a while to make progress on this, as it goes into an internal review process that is low priority for the GF team.

**Comment by davelab6** [comment]
Reassigning to me to deal with the policy issue.
